src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/EnvelopeMapperTest.java

```diff
@@ -17,46 +17,69 @@
 
 public class EnvelopeMapperTest {
+    private static final String METAFILE_BASE_PATH = "/metafiles/valid/";
 
     @Test
-    public void should_map_zip_envelope_correctly() throws Exception {
-        InputEnvelope zipEnvelope = getEnvelopeFromMetafile();
+    public void should_map_envelope_fields_payments_andnon_scannable_items_correctly() throws Exception {
+        InputEnvelope zipEnvelope = getEnvelopeFromMetafile(METAFILE_BASE_PATH + "no-scannable-items.json");
+        String container = "container1";
+
+        Envelope dbEnvelope = EnvelopeMapper.toDbEnvelope(zipEnvelope, container, Optional.empty());
+
+        assertSameFields(dbEnvelope, zipEnvelope);
+
+        assertSamePayments(dbEnvelope, zipEnvelope);
+        assertThat(dbEnvelope.getScannableItems()).isEmpty();
+        assertSameNonScannableItems(dbEnvelope, zipEnvelope);
+
+        assertDbEnvelopeSpecificProperties(container, dbEnvelope);
+    }
+
+    @Test
+    public void should_map_zip_envelope_with_scannable_items_without_sscs1_item_correctly() throws Exception {
+        InputEnvelope zipEnvelope = getEnvelopeFromMetafile(METAFILE_BASE_PATH + "no-sscs1-scannable-items.json");
         String container = "container1";
         OcrValidationWarnings ocrValidationWarnings = new OcrValidationWarnings(
             zipEnvelope.scannableItems.get(0).documentControlNumber,
-            Arrays.asList("warning 1", "warning 2")
+            asList("warning 1", "warning 2")
         );
 
         Envelope dbEnvelope = EnvelopeMapper.toDbEnvelope(zipEnvelope, container, Optional.of(ocrValidationWarnings));
 
-        assertThat(dbEnvelope.getCaseNumber()).isEqualTo(zipEnvelope.caseNumber);
-        assertThat(dbEnvelope.getPreviousServiceCaseReference()).isEqualTo(zipEnvelope.previousServiceCaseReference);
-        assertThat(dbEnvelope.getPoBox()).isEqualTo(zipEnvelope.poBox);
-        assertThat(dbEnvelope.getJurisdiction()).isEqualTo(zipEnvelope.jurisdiction);
-        assertThat(dbEnvelope.getDeliveryDate()).isEqualTo(zipEnvelope.deliveryDate);
-        assertThat(dbEnvelope.getOpeningDate()).isEqualTo(zipEnvelope.openingDate);
-        assertThat(dbEnvelope.getZipFileCreateddate()).isEqualTo(zipEnvelope.zipFileCreateddate);
-        assertThat(dbEnvelope.getZipFileName()).isEqualTo(zipEnvelope.zipFileName);
-        assertThat(dbEnvelope.getClassification()).isEqualTo(zipEnvelope.classification);
-
-        assertSamePayments(dbEnvelope, zipEnvelope);
         assertSameScannableItems(dbEnvelope, zipEnvelope);
-        assertSameNonScannableItems(dbEnvelope, zipEnvelope);
+        assertSameOcrData(dbEnvelope, zipEnvelope);
+        assertDocumentTypes(dbEnvelope, CHERISHED, OTHER, DocumentType.COVERSHEET);
         assertRightScannableItemsHaveWarnings(dbEnvelope, ocrValidationWarnings);
+    }
 
-        // properties specific to DB envelope
+    @Test
+    public void should_map_zip_envelope_with_scannable_items_with_sscs1_item_correctly() throws Exception {
```

Comment:
How about:
* naming it `should_map_document_types_and_subtypes_correctly()`
* including all existing document types in the test json
* verifying document types along with subtypes?

Reply:
ok